src/toolbars/AudioSetupToolBar.h

```diff
@@ -0,0 +1,97 @@
+/**********************************************************************
+
+  Audacity: A Digital Audio Editor
+
+  DeviceToolbar.h
+
+  Dominic Mazzoni
+
+**********************************************************************/
+
+#ifndef __AUDACITY_AUDIO_SETUP_TOOLBAR__
+#define __AUDACITY_AUDIO_SETUP_TOOLBAR__
+
+#include <optional>
+#include <vector>
+#include "ToolBar.h"
+#include "Observer.h"
+
+enum class DeviceChangeMessage : char;
+
+class wxCommandEvent;
+class wxBoxSizer;
+class wxSize;
+class wxPoint;
+class wxMenu;
+class wxString;
+struct DeviceSourceMap;
+
+class AButton;
+class AudacityProject;
```

Comment:
Maybe you copied and pasted.  I like to keep the #include directives and forward declarations minimal.  I think you can eliminate:

<vector>
class wxCommandEvent;
class wxBoxSizer;
class wxSize;
class wxPoint;
class wxMenu;
class wxString;
class AButton;
class AudacityProject;